dev/ansible/03_run_tests.yml

```diff
@@ -30,16 +30,17 @@
 
 - name: Check if intention allow exists for minio-local and minio
   shell: consul intention match -source minio-local
-  register: intention_exist
+  register: intention_exist_1
   when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
   tags: test
 
 - name: Consul create intention allow for minio-local and minio
-  shell: nohup consul intention create minio-local minio
-  register: test_output_intention
-  when: intention_exist is defined and intention_exist != {}
+  shell: consul intention create minio-local minio
+  register: register_intention_2
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny' and intention_exist_1 != {}
   environment:
     CONSUL_HTTP_TOKEN: "master"
+  failed_when: register_intention_2.rc > 0
```

Comment:
I remember that I push to add this line, but I evaluated again and it might be either overkill (the default behaviour for `shell` fail if result_code != 0) or even misleading (in case result_code is `-1`)